### PR TITLE
Resolve issue #449

### PR DESCRIPTION
**Pull request by cawerkenthin**
Address changes from issue #449 - Can Waived contain a Score?

### CODE REVIEW

File: cmi5_spec.md [hunk 1]
```diff
@@ -787,6 +787,14 @@ Example JSON:
 
 A score is not required to be reported.  If a score is reported by an AU, the verb MUST be consistent with "masteryScore" (if defined for the AU in the Course Structure).
 
+The score property of the result MAY be set in the following cmi5 defined statements:
+
+- Passed
+- Failed
+- Waived
```
Comment (brianjmiller) [inline cmi5_spec.md]: I believe the ultimate decision from the call was that Waived **does not** allow `score`, though that wasn't captured in the minutes. The reasoning given by @andyjohnson (which the group was good with) was that score would be captured via other statements from the activity causing waiver, whatever that may be.

File: cmi5_spec.md [hunk 2]
```diff
@@ -796,22 +804,26 @@ A score is not required to be reported.  If a score is reported by an AU, the ve
...
 
 <a name="completion"></a>
 ###9.5.3 Completion
-The completion property of the result MUST be set to true for statements having the following cmi5 defined verbs:
+The completion property of the result MUST be set to true for the following cmi5 defined statements:
 
 - Completed
 - Waived
 
+Other cmi5 defined statements MUST NOT include the success property.
```
Comment (brianjmiller) [inline cmi5_spec.md]: Looks like copy+paste, says "success" but this one should be "completion".

**Comment by brianjmiller** [comment]
I believe this still needs:

> Other cmi5 defined statements MUST NOT include the duration property.

For duration.

Also should the properties be quoted in these cases, I don't remember how consistent the rest of the doc is (though I'd like to see it made so).

File: cmi5_spec.md (hunk 1 above)
Comment (bscSCORM) [inline cmi5_spec.md]: That's my recollection as well.

**Comment by MrBillMcDonald** [comment]
Per May 20th Meeting will merge this request and add the following:

> **9.5.4 Duration**
>
> The duration property is an ISO 8601 formatted time value required in certain statements as defined in this section.  Other cmi defined statements MAY include the duration property.
> 

To fully resolve this issue

**Pull request merged**